apps/system/utils/netcmd.c

```diff
@@ -95,6 +95,95 @@ struct ifconfig_cmd_info_s {
 extern int netdb_main(int argc, char *argv[]);
 #endif
 
+int getIp(void)
+{
+	struct ifreq *ifr;
+	struct sockaddr_in *sin;
+	struct sockaddr *sa;
+	struct ifconf ifcfg;
+	int ret = -1;
+	int fd;
+	int numreqs = 3;
```

Comment:
There are `numreqs` and `num_nic`.
One has `_`, but one doesn't.
It is better to match the variable naming rule for clean code.